### PR TITLE
Update fields in destination-connectors specifications: mqtt, mysql, postgres, pulsar, redis, redshift, s3, scylla

### LINKED ISSUE #8748: Update fields in destination-connectors specifications: mqtt, mysql, postgres, pulsar, redis, redshift, s3, scylla
Add title and description properties to each field in destination-connectors specifications:
destination-mqtt
destination-mysql
destination-postgres
destination-pulsar
destination-redis
destination-redshift
destination-s3
destination-scylla

[EPIC] Add title and description properties to each field in connectors specifications and order input fields appropriately #8324

### PR DESCRIPTION
**Pull request by karinakuz**
## What

Add title and description properties to each field in destination-connectors specifications:
destination-mqtt
destination-mysql
destination-postgres
destination-pulsar
destination-redis
destination-redshift
destination-s3
destination-scylla

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</p>
</details>

### CODE REVIEW

File: airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json [hunk 1]
```diff
@@ -22,22 +22,23 @@
         "title": "Endpoint",
         "type": "string",
         "default": "",
-        "description": "This is your S3 endpoint url.(if you are working with AWS S3, just leave empty).",
+        "description": "URL to S3, if using AWS S3 just leave blank.",
```
Comment (yevhenii-ldv) [inline airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json:25]:
```suggestion
        "description": "URL to S3 (if using AWS S3 just leave blank).",
```

File: airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json [hunk 2]
```diff
@@ -22,22 +22,23 @@
...
         "examples": ["http://localhost:9000"]
       },
       "s3_bucket_name": {
-        "title": "S3 Bucket Name",
+        "title": "Bucket Name",
         "type": "string",
-        "description": "The name of the S3 bucket.",
+        "description": "The name of the S3 bucket o sync data into.",
```
Comment (yevhenii-ldv) [inline airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json:31]:
```suggestion
        "description": "The name of the S3 bucket to sync data into.",
```

File: airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json [hunk 3]
```diff
@@ -134,7 +135,7 @@
                     }
                   },
                   {
-                    "title": "bzip2",
+                    "title": "BZIP2",
```
Comment (yevhenii-ldv) [inline airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json:138]:
```suggestion
                    "title": "bzip2",
```
These types are always in lower case, I don't think it is necessary to put them in upper case.

File: airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json [hunk 4]
```diff
@@ -145,7 +146,7 @@
                     }
                   },
                   {
-                    "title": "xz",
+                    "title": "XZ",
```
Comment (yevhenii-ldv) [inline airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json:149]:
```suggestion
                    "title": "xz",
```
These types are always in lower case, I don't think it is necessary to put them in upper case.

File: airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json [hunk 5]
```diff
@@ -164,7 +165,7 @@
                     }
                   },
                   {
-                    "title": "zstandard",
+                    "title": "Zstandard",
```
Comment (yevhenii-ldv) [inline airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json:168]:
```suggestion
                    "title": "zstandard",
```
These types are always in lower case, I don't think it is necessary to put them in upper case.

File: airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json [hunk 6]
```diff
@@ -173,23 +174,23 @@
...
+                        "title": "Include Checksum",
                         "description": "If true, include a checksum with each data block.",
                         "type": "boolean",
                         "default": false
                       }
                     }
                   },
                   {
-                    "title": "snappy",
+                    "title": "Snappy",
```
Comment (yevhenii-ldv) [inline airbyte-integrations/connectors/destination-s3/src/main/resources/spec.json:193]:
```suggestion
                    "title": "snappy",
```
These types are always in lower case, I don't think it is necessary to put them in upper case.

File: docs/integrations/destinations/postgres.md [hunk 7]
```diff
@@ -84,6 +84,7 @@ Therefore, Airbyte Postgres destination will create tables and schemas using the
 
 | Version | Date | Pull Request | Subject |
 |:--------| :--- | :--- | :--- |
+| 0.1.14 | 2021-12-14 | [8791](https://github.com/airbytehq/airbyte/pull/8791) | Update fields in destination-connectors specifications  |
```
Comment (yevhenii-ldv) [inline docs/integrations/destinations/postgres.md:87]:
```suggestion
| 0.3.14 | 2021-12-14 | [8791](https://github.com/airbytehq/airbyte/pull/8791) | Update fields in destination-connectors specifications  |
```

File: docs/integrations/destinations/redshift.md [hunk 8]
```diff
@@ -118,6 +118,7 @@ All Redshift connections are encrypted using SSL
 
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| 0.1.22 | 2021-12-14 | [8791](https://github.com/airbytehq/airbyte/pull/8791) | Update fields in destination-connectors specifications  |
```
Comment (yevhenii-ldv) [inline docs/integrations/destinations/redshift.md:121]:
```suggestion
| 0.3.22 | 2021-12-14 | [8791](https://github.com/airbytehq/airbyte/pull/8791) | Update fields in destination-connectors specifications  |
```

**Review by yevhenii-ldv — APPROVED** [review]

**Comment by sherifnada** [comment]
won't be able to review, pls ask for review from someone else

**Review by igrankova — APPROVED** [review]

**Review by misteryeo — APPROVED** [review]

**Pull request closed without merging**